buffer/src/main/java/io/netty/buffer/PooledUnsafeHeapByteBuf.java

```diff
@@ -129,6 +129,34 @@ protected void _setLongLE(int index, long value) {
     }
 
     @Override
+    public ByteBuf setZero(int index, int length) {
+        if (PlatformDependent.javaVersion() >= 7) {
```

Comment:
@nmittler and just return `false` if we need to fallback to the "slow" path ? I guess this may be a good idea.